bin/infrasim-init

```diff
@@ -8,8 +8,8 @@ from infrasim.ipmi import get_ipmi
 from infrasim.qemu import get_qemu
 import netifaces
 
-INFRASIM_TEMP_CONF = "/usr/local/etc/infrasim/conf/infrasim.conf"
```

Comment:
This is a temporary configuration file defined in [jinja template](http://jinja.pocoo.org/docs/dev/), installed by 

    python setup.py install

With infrasim-init, it will be rendered to a runtime configuration and write to 

    INFRASIM_CONF = "/etc/infrasim/infrasim.yml"